ci-scripts/dockerfiles/tutorials.dockerfile

```diff
@@ -0,0 +1,36 @@
+#
+# Execute this from the top-level ReFrame source directory
+#
+
+
+FROM reframehpc/rfm-ci-base:lmod
```

Comment:
Is `rfm-ci-base:lmod` built using [Lmod-base.dockerfile](https://github.com/eth-cscs/reframe/blob/master/ci-scripts/dockerfiles/Lmod-base.dockerfile) ?

Maybe out of scope here but it would be nice to have a pointer to the dockerfile used.
Nvidia is always pointing to the dockerfiles they are using. For example: https://hub.docker.com/r/nvidia/cuda -> Ubuntu 20.04

Reply:
@jgphpc The docker files we have in Dockerhub are only for the CI. They are not meant for supplying ReFrame.